PIL/JpegImagePlugin.py

```diff
@@ -593,7 +593,7 @@ def _save(im, fp, filename):
 
     info = im.encoderinfo
 
-    dpi = info.get("dpi", (0, 0))
+    dpi = [int(x) for x in info.get("dpi", (0, 0))]
```

Comment:
I would round to nearest integer.

Ideally it would round to the nearest integer dpi or the nearest integer "dot per centimeter"